docker/config-webhook.example.py

```diff
@@ -0,0 +1,16 @@
+GIT_SSH_KEY = "/etc/opt/topology/github"
+
+TOPOLOGY_DATA_DIR   = "/data/webhook/topology"
+CONTACT_DATA_DIR    = "/data/webhook/contact"
+
+WEBHOOK_STATE_DIR   = "/data/webhook/state"
+WEBHOOK_DATA_DIR    = "/data/webhook/topology.git"
+WEBHOOK_DATA_REPO   = "https://github.com/opensciencegrid/topology"
+WEBHOOK_DATA_BRANCH = "master"
+WEBHOOK_SECRET_KEY  = "/etc/opt/topology/github_webhook_secret"
+WEBHOOK_GH_API_USER = "osg-bot"
+WEBHOOK_GH_API_TOKEN= "/etc/opt/topology/github_access_token"
+
+INSTANCE_NAME = "ITB webhook instance" # leave blank for production
```

Comment:
I guess this means we're calling this ITB for now?

Reply:
I left the ITB instance name there as a placeholder. As you said, the secrets and config files would come from the k8s/flux config. (Or from the local filesystem if you want to use docker for development.)